kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -20,7 +20,6 @@ window.ContentAreaView = BaseView.extend({
     show_view: function(view) {
 
         // hide any messages being shown for the old view
```

Comment:
I'm just gonna pop that clear_messages back in, should fix up the persistent message when navigating to a content page.